mesonbuild/compilers/compilers.py

```diff
@@ -946,6 +950,8 @@ def gnulike_default_include_dirs(compiler, lang):
                 break
             else:
                 paths.append(line[1:])
+    if 0 == len(paths):
```

Comment:
it isn't very pythonic to write your test in reverse order because you are not allowed to assign in an if statement anyway. So there is no risk of writing `if a = 0:` instead of `if a == 0:`

Reply:
Right you are :)